pages/index.tsx

```diff
@@ -45,58 +47,81 @@ export default function Home() {
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
-      <main className="bg-transparent min-w-screen min-h-screen ">
-        <div className="flex items-center pb-10">
-          <div className="mx-auto px-3 absolute">
-            <Image src={logo} width="50px" height="50px" alt="CRUD" className="logo" />
-          </div>          
-          <h1 className="mx-auto px-4 py-5 bg-gray-700 text-3xl font-extrabold text-left min-w-full max-w-full pl-20">
-            <span className="text-white-rose">Team Manager</span>
-          </h1>
-          <button
-              onClick={handler}
-              className= " bg-teal-2 text-white-rose flex items-center h-14 w-14 px-3 right-6 rounded hover:bg-green-1 duration-500 absolute"
-            ><span className="px-1">
-                <BiUserPlus size={23}></BiUserPlus>
-              </span>              
-            </button>
-        </div>
-
-        <div className=" flex justify-between py-5 mx-auto w-11/12">
-          <div className="left flex gap-3">
-            
-          </div>
-          {deleteId ? DeleteComponent({ deleteHandler, cancelHandler }) : <></>}
-        </div>
-
-        {/* collapse form */}
-        {visible ? <Form /> : <></>}
-
-        {/* table */}
-        <div className="overflow-auto w-11/12 mx-auto px-3 bg-white-rose">
-          <Table />
-        </div>
-
-        <div className="mx-auto pt-48 px-5 ">
-          <AppFooter />
+      <motion.section
+        initial={{ opacity: 0 }}
+        animate={{ opacity: 1 }}
+        transition={{ ease: "easeInOut", duration: 0.5, delay: 0.1 }}
+        className="">
+        <div className="background bg-lightBgPage dark:bg-darkBgPage2">
+          <main className="min-w-screen min-h-screen">
+            <div className="flex items-center pb-10">
+              <div className="mx-auto px-3 absolute"> <a href={URL}>
+                <Image src={logo} width="50px" height="50px" alt="Logo" className="logo" /></a>
+              </div>
+              <h1 className="bg-headerLight dark:bg-headerDark mx-auto px-4 py-5 lg:text-3xl md:text-xl text-left min-w-full max-w-full pl-20 shadow-md shadow-slate-900">
+                <span className="text-white-rose">Team Manager</span>
+              </h1>
+
+              <button
+                onClick={handler}
+                className="bg-blue-01 dark:bg-gray-600 hover:bg-blue-02 hover:dark:bg-gray-700 text-white-rose flex items-center h-10 w-10 pl-1 right-20 rounded absolute">
+                <span className="tooltip"> <span className="tooltiptext">Add Member</span>
+                  <BiUserPlus size={34}></BiUserPlus>
+                </span>
+              </button>
+
+              <ThemeButton />
+
+            </div>
+
+            <div >
+              {deleteId ? DeleteComponent({ deleteHandler, cancelHandler }) : <></>}
+            </div>
```

Comment:
I somehow highly doubt this wrapping `<div>` needs to exist if it only contains an empty fragment `<></>`

Reply:
In this case I would simply use a shortcircuit operator:

```suggestion
            
              {deleteId && (
                <div >
                  DeleteComponent({ deleteHandler, cancelHandler })
                </div>
              )}
```

Reply:
didn't knew this syntax! thanks